mysite/app/views.py

```diff
@@ -348,6 +351,37 @@ def tilbud_liste_bookingsjef(request):
     else:
         return redirect('dashboard')
 
+
+
+def vurder_marked(request):
+    user = request.user
+    if not request.user.is_authenticated():
+        return render(request, 'registration/login.html', {})
+
+    rolle = user.profile.role
+    if rolle == 'bookingsjef':
```

Comment:
Kunne kun hatt en sjekk for current_scene not none og current_genre not none

Reply:
Ja, kunne nok gjort det litt bedre, men bare gjør det hvis du har tid. Fungerer det nå så får brute-force være brute-force.